doc/modules/cassandra/pages/developing/cql/SASI.adoc

```diff
@@ -9,7 +9,7 @@ has superior performance in cases where queries would previously require
 filtering. In achieving this performance, SASI aims to be significantly
 less resource intensive than existing implementations, in memory, disk,
 and CPU usage. In addition, SASI supports prefix and contains queries on
-strings (similar to SQL's `LIKE = "foo*"` or `LIKE = "*foo*"'`).
+strings (similar to SQL's ``LIKE = "foo\*"`` or ``LIKE = "*foo*"`` ).
```

Comment:
Weirdly enough, my IntelliJ renderer is rendering the `` ` `` (single) and the `` ` ` ``(double) are rendered the same. Is that expected? If that's the case, we should keep consistency (at line 4, I can see the word `Index` wrapped by a single `

Reply:
![SASI_1](https://github.com/user-attachments/assets/ad53c4ab-c604-4ad8-accd-6da8b5abadcd)

fix renders properly after antora build